tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -480,6 +480,110 @@ public function test_payment_fields_outputs_fields() {
+	 * @param bool   $expect_selector  Whether the currency selector div should appear in the output.
+	 */
+	public function test_payment_fields_renders_currency_selector_conditionally(
+		bool $oc_enabled,
+		bool $valid_oc_page,
+		bool $feature_flag,
+		string $adaptive_pricing,
+		bool $expect_selector
+	): void {
+		// The gateway exposes is_adaptive_pricing_supported() as a protected instance method,
+		// allowing us to mock it directly without depending on the full settings/API stack.
+		$show_adaptive_pricing = $oc_enabled && $valid_oc_page && $feature_flag && 'yes' === $adaptive_pricing;
```

PHPMD (2.15.0)

[error] The variable $show_adaptive_pricing is not named in camelCase. (undefined)

(CamelCaseVariableName)